main.js

```diff
@@ -0,0 +1,86 @@
+console.log('aloha!');
+// ================================================================
+// when a user clicks "start" button, they should see
+// a counter decreasing from 25 minutes
+// ================================================================
+var timer = document.body.querySelector('div');
+var mins = 25; // total amount of minutes to countdown
+var secs = mins * 60; // coverts total countdown minutes to seconds
+var timerMins = 0; //
+var timerSecs = 0;
+var timerId = null;
+
+// create a function that starts at a given number
+// and counts down by 1 until it reaches 0
+var countDown = function () {
+  timerMins = Math.floor(secs / 60);
+  timerSecs = Math.floor(secs % 60);
+  // in order to keep clock format (mm:ss),
+  // when single mins/secs, print "07" insetad of "7"
+  if (timerSecs <= 9)
+    timerSecs = "0" + timerSecs;
+  secs--;
+  console.log(secs);
+  if (secs < 0){
+    clearInterval(timerId);
+    alert("Take a break, fool!");
+  }
+  //display timer as "mm:ss"
+  timer.innerHTML = timerMins + ":" + timerSecs;
```

Comment:
Nice work breaking the time into minutes and seconds and only setting one timer.